lit.css

```diff
@@ -4,7 +4,16 @@
 * Font set to: Nunito
 * Use ctrl-f to change these
 */
-
+:root {
+    --primary-font: Nunito;
+    --primary-color: #212121;
+    --secondary-color: #dbdbdb;
+    --accent-color: #234567;
+    --link-color: gold;
+    --border-color: green;
+    --primary-button-bg: gold;
```

Comment:
Hey, could you change the "gold" back to the original hexadecimal value of #FA0? #FA0 saves one extra byte with gzip for some reason. If you want, you could add a comment after the "#FA0" saying "gold".

Other than that, I think that this is a great idea and will merge as soon as I figure out the PostCSS stuff!